src/libmain/progress-bar.cc

```diff
@@ -180,10 +180,8 @@ class ProgressBar : public Logger
             auto machineName = getS(fields, 1);
             if (machineName != "")
                 i->s += fmt(" on " ANSI_BOLD "%s" ANSI_NORMAL, machineName);
-            auto curRound = getI(fields, 2);
-            auto nrRounds = getI(fields, 3);
-            if (nrRounds != 1)
-                i->s += fmt(" (round %d/%d)", curRound, nrRounds);
+            getI(fields, 2); // Used to be curRound, but build repetition hasn't really worked in ages
+            getI(fields, 3); // Used to be nrRounds, but ^
```

Comment:
Warn if we get something we don't expect?

Reply:
Still on the fence about what to do about that

Reply:
I decided this is good, because it is tantamount to asserting that we took a certain control flow in the old code.
```suggestion
            // Used to be `nrRounds` and `curRound`, but build repetition hasn't really worked in ages.
            // `nrRounds` being something other than 1 indicated repetition before, so we assert that it is 1.
            assert(getI(fields, 3), 1);
            // `curRounds` was ignored if `nrRounds` was `1` (no repetition) so we also ignore it.
            getI(fields, 2);
```